workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_home_cards.scss

```diff
@@ -0,0 +1,43 @@
+.home-cards {
+	display: grid;
+	gap: 48px;
+	grid-template-columns: repeat(auto-fit, 254px);
+	grid-template-rows: 130px;
+	max-width: 948px;
+	padding: 0 1.3125rem;
+
+	@media (max-width: 1200px) {
+		grid-template-columns: repeat(2, 254px);
+	}
+
+	@media (max-width: 576px) {
+		grid-template-columns: repeat(1, 254px);
+	}
+
+	.product-card {
+		background-color: #fff;
+		border: 1px solid rgba(9, 16, 24, 0.12);
+		border-radius: 0.25rem;
+		box-shadow: 0 6px 15px -6px rgb(9 16 29 / 6%);
```

Comment:
Some of this card css is the same for the landing page cards and these new cards - can you find a way to just define it in one place?

Reply:
I extracted these styles for home cards and landing page.